orders/views.py

```
@@ -11,11 +11,13 @@
 from sparky_utils.response import service_response
 from utils.utils import generate_ref, PaystackSDK, get_client_ip
 
-from app.models import Cart, Product, CartItem
+from app.models import Cart, ExchangeRate, Product, CartItem
 from app.tasks import send_email_async
 import hmac
 import hashlib
 from utils.constants import PaymentStatus
+from utils.utils import get_client_ip, get_country_currency_from_ip
```

Ruff (0.11.9)

Redefinition of unused `get_client_ip` from line 12

Remove definition: `get_client_ip`

(F811)